tests/processor/test_stream_task.py

```diff
@@ -0,0 +1,53 @@
+"""
+StreamTask tests
+"""
+
+from unittest.mock import Mock, patch
+
+import pytest
+from confluent_kafka.cimpl import KafkaError, KafkaException
+
+from winton_kafka_streams import kafka_config
+from winton_kafka_streams.errors.task_migrated_error import TaskMigratedError
+from winton_kafka_streams.processor import TopologyBuilder
+from winton_kafka_streams.processor._stream_task import StreamTask
+from winton_kafka_streams.processor.task_id import TaskId
+
+taskMigratedErrorCodes = [KafkaError.ILLEGAL_GENERATION,
+                          KafkaError.UNKNOWN_MEMBER_ID,
+                          KafkaError.REBALANCE_IN_PROGRESS,
+                          47 # INVALID_PRODUCER_EPOCH - not supported in all versions for Conluent Kafka so just use the explicit code in this test
+                          ]
```

Comment:
How recent does confluent_kafka need to be?

We're pretty bleeding edge everywhere else, so could be here as well.

Reply:
It seems the version on Travis is older - that's where the tests failed.